evm_loader/program/src/instruction/transaction_cancel.rs

```diff
@@ -43,7 +43,11 @@ fn validate(accounts: &Accounts, transaction_hash: &[u8; 32]) -> ProgramResult {
     Ok(())
 }
 
-fn execute<'a>(program_id: &'a Pubkey, accounts: Accounts<'a>) -> ProgramResult {
+fn execute<'a>(
+    program_id: &'a Pubkey,
+    accounts: Accounts<'a>,
+    blocked_accounts: &BlockedAccounts,
+) -> ProgramResult {

```

Comment:
You can rewrite this function without `ProgramAccountStorage`. It will be simplified in this case.

Reply:
Done.